cli/cmd/secrets.go

```diff
@@ -76,13 +79,11 @@ func createSecret() *cobra.Command {
 					return err
 				}
 				defer in.Close()
-
 			}
 			content, err := ioutil.ReadAll(in)
 			if err != nil {
 				return fmt.Errorf("Error reading content from %q: %v", file, err)
```

Comment:
Same here
```suggestion
				return fmt.Errorf("error reading content from %q: %v", file, err)
```

Reply:
Not a blocker, just a suggestion: I usually try to avoid "error" as part of the error message (it should be clear there's an error). Also could consider using `%w` to wrap the error, in case we want to do further processing (although not likely), for example;

```suggestion
				return fmt.Errorf("failed to read content from %q: %w", file, err)
```